pom.xml

```diff
@@ -186,7 +186,7 @@
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>3.10.1</version>
                     <configuration>
-                        <release>11</release>
+                        <release>17</release>
```

Comment:
I believe this will force us to update the CLI as well, and make Java 17 a runtime requirement for the CLI. If so, there should be a follow up ticket to update the CLI, as well as the doc and UI where ever we list Java 11 as the minimum Java.